ansible/library/port_alias.py

```diff
@@ -112,7 +112,7 @@ def get_portmap(self):
                     aliases.append(alias)
                     portmap[name] = alias
                     aliasmap[alias] = name
-                    if speed_index != -1:
+                    if (speed_index != -1) and (len(mapping) > speed_index):
```

Comment:
>if [](start = 20, length = 2)

Otherwise, should we throw?

Reply:
I think we shouldn't throw here, if the last column of the port_config.ini is "speed", then it's OK to have some ports left w/o speed explicitly specified(at least in our practice we have this case).  In this case,  means the speed on this port doesn't matter, in the later on minigraph generate process, will assign a default speed value to it.